modules/tensor_mechanics/doc/content/modules/tensor_mechanics/2d_elastic_waves.md

```diff
@@ -0,0 +1,77 @@
+# Modeling Elastic Waves in Frequency Domain
+
+Frequency domain analyses can be advantageous over time domain counterpart in several cases, for example, when the frequency spectrum of a signal consists of a few frequencies, or, when it is needed to have a better control of the numerical dispersion.
```

Comment:
The introduction should be something like this to let analyst know what can be done with frequency domain simulatiosn. 

The following example presents a frequency domain analysis done in the MOOSE Tensor Mechanics module.  A frequency domain analysis provides the structural response at a discrete frequency where each frequency is the solution of an independent steady state simulation.  

Frequency domain analysis is often used to determine a frequency response function (FRF).  An FRF describes the relationship between an input and output in the frequency domain.  The input location is where the input load is applied, this is usually a Fourier transform of a measured transient load function, and the output location is where the response like acceleration is measured.  Peaks on the FRF are the natural modes of the system.  

Frequency domain analyses can be advantageous over time domain counterpart in several cases, for example, when the frequency spectrum of a signal consists of a few frequencies, or, when it is needed to have a better control of the numerical dispersion.